Tasks/Gordeenko/Task2/Task2.py

```diff
@@ -0,0 +1,98 @@
+# Реализовать текстовый вывод времени (выбор пользователя) - текущее системное время или ручной ввод hh:mm 
+
+import datetime
+time = datetime.datetime.now()
+
+x = int(input("Please choose an option: automatic (0) or user (1) time entry:"))
+
+if x == 0:
+    print(f"{time.hour} : {time.minute}")
+if x == 1:
+    time_aut_hour = int(input("Please enter the time in hour and minute format (hh):"))
+    time_aut_minute = int(input("Please enter the time in hour and minute format (mm):"))
```

Comment:
it should be a single input in the "hh:mm" format

Reply:
Update single input in the "hh:mm" format